libs/full/parcelport_lci/include/hpx/parcelport_lci/sender_connection.hpp

```diff
@@ -291,7 +291,9 @@ namespace hpx::parcelset::policies::lci {
             buffer_.data_point_.time_ =
```

Comment:
Should this be disabled as well?

Reply:
I am hesitant because if we disable all the `buffer_.data_point_` related lines, there will be too many `ifdef` all over the code. I am kind of hoping the compiler's dead code elimination is smart enough to remove these lines.

Reply:
> I am hesitant because if we disable all the `buffer_.data_point_` related lines, there will be too many `ifdef` all over the code. I am kind of hoping the compiler's dead code elimination is smart enough to remove these lines.

Right. I would at least suggest removing the invocations of the timers. Those introduce a bit of avoidable overhead. In general, I wouldn't rely on the compiler to be smart enough to eliminate unneeded code.

Reply:
Okay, let me think about it. Maybe a better way is to create a set method for each field of the data point so that we can directly replace the methods with NULL methods.

Reply:
Eventually, I fell back to the trivial solution. It's hard to find an easy way to encapsulate the use of timer within a single method of data_point.